RoadMap_Movie/RoadMap_Movie/MainModule/View/MainScreenViewController.swift

```diff
@@ -63,14 +63,24 @@ final class MainScreenViewController: UIViewController {
         return button
     }()
 
-    private let tableView = UITableView()
-
     // MARK: - Private properties
 
+    private let networkService = MovieNetworkService()
+    private let tableView = UITableView()
     private let sessionConfiguration = URLSessionConfiguration.default
     private let decoder = JSONDecoder()
     private let session = URLSession.shared
-    private var movies: Results?
+    private var movies: [Movie] = [] {
```

Comment:
```suggestion
    {
```
массив надо удалить тут